content/platforms/release-notes/k8s-6-0-6-24-2020-08.md

```diff
@@ -0,0 +1,89 @@
+The Redis Enterprise K8s 6.0.6-24 release is a *maintenance release* on top of [6.0.6-23](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.0.6-23) providing support for the latest [Redis Enterprise Software release 6.0.6-39](https://docs.redislabs.com/latest/rs/release-notes/rs-6-0-may-2020/) and includes several bug fixes.
+
+## Overview
+
+This release of the operator provides:
+
+ * The latest release version of the operator
+ * Various bug fixes
+
+
+Follow the instructions in ["Upgrading a Redis Enterprise Cluster in Operator-based Architecture"]({{< relref "/platforms/kubernetes/upgrading-kubernetes-with-operator.md" >}}) to upgrade to this Kubernetes operator release.
+
+
+## Images
+
+ * **Redis Enterprise** - redislabs/redis:6.0.6-39 or redislabs/redis:6.0.6-39.rhel7-openshift
+ * **Operator** - redislabs/operator:6.0.6-24
+ * **Services Rigger** - redislabs/k8s-controller:6.0.6-24 or redislabs/services-manager:6.0.6-24 (on the RedHat registry)
+
+## Important fixes
+
+ * A fix for database observability where after 24 hours after creation or update, the controller was unable to observe the database (RED46149)
+ * A fix for a log collector crash on Windows when pods were not running (RED45477)
+
+## Known limitations
+
+### CrashLoopBackOff causes cluster recovery to be incomplete  (RED33713)
+
+When a pod status is CrashLoopBackOff and we run the cluster recovery, the process will not complete. The workaround is to delete the crashing pods manually and recovery process will continue.
+
+### Long cluster names cause routes to be rejected  (RED25871)
+
+A cluster name longer than 20 characters will result in a rejected route configuration as the host part of the domain name exceeds 63 characters. The workaround is to limit cluster name to 20 characters or less.
+
+### Cluster CR (REC) errors are not reported after invalid updates (RED25542)
+
+A cluster CR specification error is not reported if two or more invalid CR resources are updated in sequence.
+
+### An unreachable cluster has status running (RED32805)
+
+When a cluster is in an unreachable state the state is still running instead of being reported as an error.
+
+### Readiness probe incorrect on failures (RED39300)
+
+STS Readiness probe doesn't mark a node as not ready when rladmin status on the node fails.
+
+### Role missing on replica sets (RED39002)
+
+The redis-enterprise-operator role is missing permission on replica sets.
+
+### Private registries are not supported on OpenShift 3.11 (RED38579)
+
+Openshift 3.11 doesn't support dockerhub private registry. This is a known OpenShift issue.
+
+### Internal DNS and Kubernetes DNS may have conflicts (RED37462)
+
+DNS conflicts are possible between the cluster mdns_server and the K8s DNS. This only impacts DNS resolution from within cluster nodes for Kubernetes DNS names.
+
+### 5.4.10 negatively impacts 5.4.6 (RED37233)
+
+Kubernetes-based 5.4.10 clusters seem to negatively impact existing 5.4.6 clusters.
```

Comment:
@alexmilowski It seemed that the word clusters was missing. Correct?

Reply:
```suggestion
Kubernetes-based 5.4.10 deployments seem to negatively impact existing 5.4.6 deployments that share a Kubernetes cluster.
```
@bmansheim better?